lib/fns.sh

```diff
@@ -197,6 +197,9 @@ function _Dbg_linespec_setup {
   typeset -ri is_function=${word[1]}
   line_number=${word[0]}
   full_filename=$(_Dbg_is_file "$filename")
+  if [[ -z "$full_filename" ]] ; then
+      full_filename=$(_Dbg_resolve_expand_filename "$filename")
```

Comment:
Is quoting needed here?

Reply:
I tested with quoting and it doesn't seem to hurt. Personally, I find the quotes within quotes confusing, and more generally the multiple substitution passes weird, but this is bash after all. 

So I've now made the change.